python_files/ml_on_iris_dataset.py

```diff
@@ -0,0 +1,192 @@
+# Iris Machine Learning Assignment 1
+# Will McGrath
+# September 8, 2020
+
+import sys
+
+import numpy as np
+import pandas as pd
+from plotly import express as px
+from plotly import graph_objects as go
+from sklearn.ensemble import RandomForestClassifier
+from sklearn.metrics import log_loss, roc_auc_score
+from sklearn.pipeline import Pipeline
+from sklearn.preprocessing import Normalizer, OneHotEncoder
+from sklearn.svm import LinearSVC
+
+
+def print_heading(title):
+    # creates headers to divide outputs
+    print("\n")
+    print("*" * 80)
+    print(title)
+    print("*" * 80)
+
+    return None
+
+
+def main():
+    # pulls in the Iris dataset from a CSV and creates a dataframe
+    print_heading("Pulling in Data and Creating Dataframe")
+    data = pd.read_csv("./data/iris.data", header=None)
+    data_df = pd.DataFrame(data).rename(
+        columns={
```

Comment:
Just a stylistic choice, but wouldn't it be better for the reader to have the full name for the column instead of having it abbreviated?